test/local/test_config_yaml.py

```diff
@@ -271,6 +271,20 @@ def test_original_syntax_scm(self):
         self.assertEqual(uri, wrap.get_uri())
         self.assertEqual({scmtype: {'local-name': local_name, 'uri': uri}}, wrap.get_legacy_yaml())
 
+        # version is a number
+        local_name = 'common_rosdeps'
+        version = 1234
+        uri = 'https://kforge.ros.org/common/rosdepcore'
+        scmtype = 'hg'
+        struct = {scmtype: {'local-name': local_name, 'version': version, 'uri': uri}}
+        expected_struct = {scmtype: {'local-name': local_name, 'version': str(version), 'uri': uri}}
+        wrap = get_path_spec_from_yaml(struct)
+        self.assertEqual(scmtype, wrap.get_scmtype())
+        self.assertEqual(scmtype, wrap.get_legacy_type())
+        self.assertTrue(isinstance(wrap.get_version(), str))
```

Comment:
You might consider using `assertIsInstance` here.

Reply:
fixed